CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+# 82.0.0
+
+In this major release, we dropped support for iOS 8 in all components. We also introduced new method and properties for AppBar, BottomSheet and NavigationDrawer.
+
+## Breaking changes
+
+iOS 8 support has been dropped in all components. This change will require application bumping minimum depolyment target to iOS 9.
```

Comment:
I would not use "bumping" since it's slang and not really appropriate for docs like this.  My suggestion:

"This release drops support for iOS 8 and the library, catalog, examples, and demos may no longer build for iOS 8 targets or may crash even if building succeeds.  Clients will need to set their minimum deployment targets to iOS 9 or later."

Reply:
Addressed in 2cb0bf1b20ef7d58d617fc09344b880ffbd376e5.